tests/difficulty/wide_difficulty.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/python
+
+import sys
+import subprocess
+
+python = sys.argv[1]
+py = sys.argv[2]
+c = sys.argv[3]
+data = sys.argv[4]
+
+first = python + " " + py + " > " + data
+second = [c, '--wide', data]
+
+try:
+  print 'running: ', first
+  subprocess.check_call(first, shell=True)
+  print 'running: ', second
```

Comment:
I had to add parentheses to these prints with Python 3.6.3. But `print time, diff` in gen_wide_data.py did work fine, so I'm left confused...

Reply:
Hmm, that's my work branch that's gone out of sync with the d128 branch, sorry about that. I'll reset from the one from the work branch.

Reply:
Looks like my comments aren't getting saved. Again: this is because d128 and my work branch got out of sync. I pushed a fresh one.

Reply:
Hmm. I seee both now. Whatever.